wrapper/src/main/java/software/amazon/jdbc/plugin/cache/DataRemoteCachePlugin.java

```diff
@@ -66,10 +70,13 @@ public DataRemoteCachePlugin(final PluginService pluginService, final Properties
     }
     this.pluginService = pluginService;
     this.telemetryFactory = pluginService.getTelemetryFactory();
-    this.hitCounter = telemetryFactory.createCounter("remoteCache.cache.hit");
-    this.missCounter = telemetryFactory.createCounter("remoteCache.cache.miss");
+    this.jdbcCachedQueryCounter = telemetryFactory.createCounter("JdbcCachedQueryCount");
+    this.jdbcCacheMissCounter = telemetryFactory.createCounter("JdbcCacheMissCount");
     this.totalCallsCounter = telemetryFactory.createCounter("remoteCache.cache.totalCalls");
```

Comment:
Probably rename this to match the naming convention of the rest of the metrics. i.e. `JdbcCacheTotalQueryCount`